### ISSUE TITLE
Revert "Prevent setting a build directory outside of your vite project"

### ISSUE THREAD

**Issue by eldr0n** [lhapaipai/vite-plugin-symfony#34]
### vite-plugin-symfony version

6.1.2

### vite-bundle version

6.1.2

### your OS, node version, Symfony version, PHP version

nixos, v20.9.0, 6.38, 8.2

### Description

The pull request https://github.com/lhapaipai/symfony-vite-dev/pull/3 prevents the usage of monorepo setups like the following: 
```
.
├── app1
│   ├── package.json
│   └── vite.config.ts
├── app2
│   ├── package.json
│   └── vite.config.ts
├── package.json
└── public
    └── build
        ├── app1
        └── app2
```

Also i dont think this plugin should prevent a config, which is already handled by vite.
```
(!) outDir /home is not inside project root and will not be emptied.
Use --emptyOutDir to override.
```

### How to reproduce

Set a build directory outside the vite project.

### Possible Solution

Revert https://github.com/lhapaipai/symfony-vite-dev/pull/3

**Comment by lhapaipai** [comment]
However, I am agree with @Huppys, a bad configuration can lead to undesirable results.
the command line option `--emptyOutDir` is not usable because it is valid with `vite build --emptyOutDir` while in our case it is also in development that we need to empty the folder. (for the entrypoints.json file).

I therefore corrected the behavior which will always display a warning if the outDir directory is not a root subfolder. The script will exit indicating the contents of the folder which will be about to be emptied. However this behavior can be forced by adding the `emptyOutDir` option in the `vite.config.js`.

```js
export default {
   build: {
     emptyOutDir: true
   }
}
```

**Comment by lhapaipai** [comment]
available from version 6.1.3
